apisix/discovery/consul/init.lua

```diff
@@ -197,21 +197,22 @@ local function get_opts(consul_server, is_catalog)
         port = consul_server.port,
         connect_timeout = consul_server.connect_timeout,
         read_timeout = consul_server.read_timeout,
+        default_args = {
+            token = consul_server.token,
+        }
     }
     if not consul_server.keepalive then
         return opts
     end
 
     if is_catalog then
```

Comment:
this conditional can be simplified

Reply:
Thx, yes of course. I just wanted to change as less as possible. I'll change it.